subsys/disk/disk_access_sdhc_spi.c

```diff
@@ -828,7 +827,6 @@ static int disk_spi_sdhc_init(struct device *dev)
 
 static struct sdhc_spi_data sdhc_spi_data_0;
 
-#ifdef DT_ZEPHYR_MMC_SPI_SLOT_0_LABEL
```

Comment:
Is this right? It seems strange to remove an `#ifdef` without also removing an `#endif`

Reply:
Actually I moved this to the beginning of the file, otherwise, there will be lots of compile warnings in case that no SPI is associated to SDHC but SPI SDHC is configured.
#ifndef DT_INST_0_ZEPHYR_MMC_SPI_SLOT_LABEL
#warning NO SDHC slot specified on board
#else
....
#endif